.github/workflows/release-published.yml

```diff
@@ -490,9 +490,9 @@ jobs:
       groupId: 'org.liquibase'
       artifactId: 'liquibase'
       version: ${{ needs.setup.outputs.version }}
-      dry_run: ${{ inputs.dry_run }}
-      dry_run_zip_url: ${{ inputs.dry_run_zip_url }}
-      dry_run_tar_gz_url: ${{ inputs.dry_run_tar_gz_url }}
+      dry_run: ${{ inputs.dry_run || 'false'}}
```

Comment:
type is boolean, should be `false` without `''`?

Reply:
yes.